src/main/java/org/gridsuite/study/server/StudyService.java

```diff
@@ -1950,6 +1950,12 @@ private BuildInfos getBuildInfos(UUID nodeUuid) {
     public void buildNode(@NonNull UUID studyUuid, @NonNull UUID nodeUuid) {
         BuildInfos buildInfos = getBuildInfos(nodeUuid);
         updateBuildStatus(nodeUuid, BuildStatus.BUILDING);
+        try {
+            buildInfos.getReportUuids().forEach(reportService::deleteReport);
+        } catch (HttpClientErrorException.NotFound e) {
```

Comment:
Remove try-catch

Reply:
OK, as the deleteReport request has ERROR_ON_REPORT_NOT_FOUND, false.

Reply:
Done